src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -555,7 +555,7 @@ impl ThinPool {
                 // Update pool space state
                 self.free_space_state = self.free_space_check(
                     usage.used_data,
-                    current_total + sectors_to_datablocks(backstore.available()) - usage.used_data,
+                    current_total - usage.used_data + sectors_to_datablocks(backstore.available()),
```

Comment:
There's no reason to reverse this order...it looks like it just crept in randomly, and it is kind of confusing.

Reply:
see 24f7a05, sorry about that, added some parens. It seemed easier to understand to group this way.